scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let FileContent = item.getAttribute('data-tagsearch-path');
+		if (FileContent) {
+
+			let matchEncrypted = await sha256(FileContent);
+			let foundFile = null;
+			for (k in apiResponses["hunkinfo"]) {
```

Comment:
You have used the `.some` operation in the bitbucket function. Maybe you can use the same here for the `foundFile` variable?